org-formation/organization.yaml

```
@@ -650,6 +651,19 @@ Organization:
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-synapsellm-prod@sagebase.org
 
+  GenAiIcDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-genaiic-dev
+      RootEmail: aws-genaiic-dev@sagebase.org
+      Alias: org-sagebase-genaiic-dev
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        AccountOwner: it@sagebase.org
+        CostCenter: NO PROGRAM / 000000
```

```suggestion
        CostCenter: NO PROGRAM / 000000
```

@milen-sage should this be BDF